index.html

```diff
@@ -14,359 +15,364 @@
       <a  class="logo" href="#">My logo</a>
     </div>
     <nav class="container">
+      
       <div class="menu-wrapper">
-        <img class="mobile-menu" src="images/Union.jpg" alt="union image">
+        <img id="mobile-menu" class="mobile-menu" src="images/Union.jpg" alt="open navigation">
       </div>
-      <ul class="nav-list">
+      <div id="nav-list-menu" class="nav-list-menu">
```

Comment:
- [ ] Your "Portfolio" link on the menu goes to the header of the page, not to the projects section.  Kindly please, correct the link so it goes to the projects section on your portfolio.

## We need to make the menu look exactly like the Figma design.

### Therefore:

- [ ]  In order to achieve the correct font weights, please include the 600 weights on your `styles.css`:

On your CSS replace this:
```css
@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;500;700&display=swap");
```

With this:
```css
@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;500;600;700&display=swap");
```

That way, you include the font-weight 600 which is required on the menu items.
- [ ] The line-height of your fonts should be 44px, and the font-weights should be 600, not 400.


- [ ] Please, align your menu items as shown in the Figma.  Look:

Your Implementation | Figma Design
:---:|:---:
![image](https://user-images.githubusercontent.com/57421823/162808808-1814b22e-3fb6-4e46-b79d-ece9f0db83ae.png) | ![image](https://user-images.githubusercontent.com/57421823/162808892-e26d59c2-3079-4c58-be32-bdeac7781e11.png)

In order to achieve this:

  - Increase the top margins/paddings so the menu items are on the same vertical level as on the Figma.
  - Increase the spacing between the menu items, so they look like on the design.
  - Increase the left spacing a little more so the menu  items are on the same horizontal level as on the Figma.

- [ ] Lastly, please move the "X" button a little lower, so it is on the same place as the Figma.



If you implement these changes, your mobile menu will look like the Figma! 💪